server/src/main/java/fr/uga/l3miage/tp1/exo2/models/NetflixUserEntity.java

```diff
@@ -0,0 +1,35 @@
+package fr.uga.l3miage.tp1.exo2.models;
+
+import fr.uga.l3miage.tp1.exo2.enums.Sex;
+
+import javax.persistence.*;
+import java.math.BigInteger;
+import java.time.LocalDate;
+
+@Entity
+@Table(name="netflix_user")
+public class NetflixUserEntity {
+    @Id
+    @Column(name = "id")
+    private BigInteger id;
+
+    @Column(name = "uuid")
+    private String uuid;
+
+    @Column(name = "lastname")
+    private String lastname;
+
+    @Column(name = "firstname")
+    private String firstname;
+
+    @Column(name = "sex")
```

Comment:
Il te manque la taille de l'enum 
```suggestion
    @Column(name = "sex", length=2 )
```